### DISCUSSION TITLE
2023 年年终总结投资篇 - forecho's Blog

### DISCUSSION THREAD

**Discussion by giscus** [forecho/hugo-blog#44] [Announcements]
# 2023 年年终总结投资篇 - forecho's Blog

引言
2023 这糟糕的一年终于要结束了，按照惯例，每年来一次投资年终总结，来看看这一年的投资表现如何。这是这系列的第五篇文章，前四篇文章：

2022 年年终总结投资篇
【深度理财 34】2021 年年终总结投资篇
【深度理财 19】2020 年年终总结投资篇
【深度理财 06】聊聊这些年我买过的股票

加一个风险提示：本篇文章仅代表我的个人观点，不构成与其他人相关的任何投资建议。在作出任何投资决定前，投资者应根据自身情况考虑投资产品相关的风险因素，并于需要时咨询专业投资顾问意见。

PS：下文中的今年是指 2023 年。
基金 && A 股
概况

今年基金收益表现还不错，总体收益 7% 左右，去年总体在收益 -13% 左右。
今年基金之所以不错得益于我的一个重要的决策，3 月份底清仓了支付宝基金，4 月底清仓了京东金融上面的国内基金，保留了几个美国基金。
全部亏损割肉了，年底回顾，清仓之后国内基金都跌了 30% 左右，而美国基金都涨了 30% 左右，这个决策是非常正确的。

https://blog.forecho.com/2023-review-investment.html

**Comment by mzihen** [comment]
写的非常详细，让我对美股也感兴趣了。这些天美股涨的很厉害，倒是港股和A股，哎……

**Comment by xyzbz** [comment]
后面就该更新奶爸的快乐生活了

**Comment by daimajia** [comment]
铁铁 你这样很容易亏完的 没几个天天赌期权的能在市场活下来😂

**Reply by forecho** [reply]
我不是纯赌期权，我是有工具查看数据的，加上自己策略和风控，已经执行半年多了，收益还不错。

纯赌最终的结果肯定是归零。

**Comment by yeung66** [comment]
新年快乐！某种程度上博主可算是我美股的引路人哈哈，祝博主在转行全职做 Trader 能够成功，也能给我们更多的激励哈哈

**Comment by helloChenLei** [comment]
今年我开始也有身份的转变了。😄
发现很多人的「自由职业」都是长期积累，直到一个契机产生，就自由了。
